From 293ff6d5bc9d3b8bfbc3c57d20d3d2d00986c168 Mon Sep 17 00:00:00 2001
From: Alexander Couzens <lynxis@fe80.eu>
Date: Sun, 26 Nov 2017 05:22:51 +0100
Subject: [PATCH] trace-cmd listen: Include limits.h to define PATH_MAX

PATH_MAX is defined by limits.h. On some system it's also
defined by dirent.h but not on musl.

Signed-off-by: Alexander Couzens <lynxis@fe80.eu>
Signed-off-by: Steven Rostedt (VMware) <rostedt@goodmis.org>
---
 trace-listen.c | 1 +
 1 file changed, 1 insertion(+)

--- a/trace-listen.c
+++ b/trace-listen.c
@@ -23,6 +23,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <getopt.h>
+#include <limits.h>
 #include <sys/types.h>
 #include <sys/socket.h>
 #include <sys/wait.h>
